tf-modules/cumulus/outputs.tf

```diff
@@ -169,3 +169,7 @@ output "ecs_cluster_name" {
 output "start_sf_queue_url" {
   value = module.ingest.start_sf_queue_url
 }
+
+output "background_queue_url" {
```

Comment:
So this is the functional change..... I'm not sure how this ties to CUMULUS-2630?

Reply:
So CUMULUS-2630 is closed, but I needed this work to generate lots of data for our dashboards.  I had this in my stashes and Katie has asked for the tools I used to make lots of data, so I thought I would put it in here as an example of a throttled workflow for discovering lots of granules.  If it's not appropriate I'm fine leaving it out.

Reply:
So I'm a bit hesitant to expose this. It's an internal queue that is throttled, but users can't control the throttling limit:

https://github.com/nasa/cumulus/blob/master/tf-modules/ingest/message_template.tf#L47

It exists solely for the purposes of us having a throttled queue built into Cumulus, which we send reingest operations to by default because we don't care about them happening very quickly: 

https://github.com/nasa/cumulus/blob/master/tf-modules/ingest/message_template.tf#L47

I'm inclined to say that for anyone (including us for testing) who wants to have their own throttled queue setup, they should implement a new queue so that they can control the execution limit:

https://nasa.github.io/cumulus/docs/data-cookbooks/throttling-queued-executions

Beyond that, I'm okay with having a  discover-granules workflow deployed with our example stacks that supports throttling by default.

Reply:
Ok, this goes along with the, if this is all wrong tell me.  I can put this back as private and create another queue in our example